Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:20.11.1
+
+WORKDIR /app
+
+COPY package*.json ./
+
+RUN npm i
+
+COPY . .
+
+ENTRYPOINT ["npm", "run", "dev"]
```

Comment:
When we first kicked off I talked about final newline settings and the like. When you don't have an empty line at the end of your file `this` is the circle error that GitHub displays!

```suggestion
ENTRYPOINT ["npm", "run", "dev"]

```

Reply:
In the event you're using multiple computers:

These are the settings I recommend 
![image](https://github.com/bcgov/nr-sustainment-capstone-2024/assets/62873746/b95b78bf-3ab6-4244-8281-faaa33b3f845)

And if you login to github through VSC you can enable settings sync :)

Reply:
^ this is good!  Seconded